pkg/sqs/client.go

```diff
@@ -182,3 +184,17 @@ func (c *Client) ReceiveData(ctx context.Context, data interface{}) (string, err
 
 	return message.ReceiptHandle, err
 }
+
+// ListQueues lists all available queues.
+func (c *Client) ListQueues(ctx context.Context) (*sqs.ListQueuesOutput, error) {
```

Comment:
I'd rather return `[]string` here to hide the existence of `*sqs.ListQueuesOutput` from the end user